config/splunk_matrix.conf

```
@@ -23,8 +23,8 @@ METRICS_MULTI = true
 INPUT_LOOKUP = true
 
 [9.1]
-VERSION = 9.1.5
-BUILD = 29befd543def
+VERSION = 9.1.6
+BUILD = a28f08fac354
 SUPPORTED = 2025-07-06
```

according to https://www.splunk.com/en_us/legal/splunk-software-support-policy.html the end of support date for Splunk is 2025-06-28. Is that correct?
```suggestion
SUPPORTED = 2025-06-28
```